src/structures/MessageEmbed.js

```diff
@@ -359,9 +359,16 @@ class MessageEmbed {
     return { name, value, inline };
   }
 
+  /**
+    * @typedef {Object} EmbedFieldData
+    * @property {StringResolvable} name The name of this field
+    * @property {StringResolvable} value The value of this field
+    * @property {?boolean} inline If this field will be displayed inline
```

Comment:
It's optional, not nullable :eyes:
```suggestion
    * @property {boolean} [inline] If this field will be displayed inline
```